lerna.json

```diff
@@ -1,7 +1,21 @@
 {
   "version": "2.4.44-alpha.12",
   "npmClient": "yarn",
-  "packages": ["packages/*"],
+  "packages": [
+    "packages/backend-core",
+    "packages/bbui",
+    "packages/builder",
+    "packages/cli",
+    "packages/client",
+    "packages/frontend-core",
+    "packages/sdk",
+    "packages/server",
+    "packages/shared-core",
+    "packages/string-templates",
+    "packages/types",
+    "packages/worker",
+    "packages/pro/packages/pro"
```

Comment:
We could consider bringing in pro as a direct dependency and ditching the lerna setup in that repo, it was setup that way for future expansion but that never materialised. Could save us from having to do any custom bootstrapping logic. 

The only thing that stands out is the offline license generation script which rely on the `offline-keys` dir at the top level. Arguably this script should be pulled out of the pro repo altogether and used in a standalone repo that pulls secrets from secrets manager instead. There's a bit of risk having it anywhere near the source code when it comes to packaging and releasing to npm.

Reply:
The current approach is:
1. Use the pro repo as it is now as a sumodule
2. Get sure everything runs fine (build, tests, pipelines, etc)
3. Publish this changes

At this point we will know that this approach works fine. So then, when the time is convenient and in sync on whatever other work is happening in pro, we will refactor pro to ditch lerna and have it as a single package

Otherwise, I will be having plenty of conflicts in pro while setting everything up in budibase repo